src/style.css

```diff
@@ -1,3 +1,96 @@
 body {
+  font-family: "Roboto", sans-serif;
   margin: 0;
 }
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  flex-direction: row;
+  max-width: 1200px;
+  margin: auto;
+}
+
+@media (max-width: 1024px) {
+  .header {
+    max-width: 1024px;
+  }
+}
+
+img {
+  margin-top: 10px;
+  margin-left: 50px;
+  width: 40px;
+  height: 40px;
+}
+
+nav {
+  margin-right: 50px;
+}
+
+.nav__list {
+  margin: 0;
+  display: flex;
+  padding: 0;
+  justify-content: space-around;
+  list-style: none;
+}
+
+.nav__item {
+  text-transform: uppercase;
+  font-size: 12px;
+  height: 60px;
+}
+
+.nav__item:not(:last-child) {
+  margin-right: 20px;
+}
+
+.nav__item__link {
+  text-decoration: none;
+  color: #000;
+  padding-top: 22.5px;
+  padding-bottom: 22.5px;
+  display: inline-block;
+}
+
+.nav__item__link:hover {
+  text-decoration: none;
+  color: #00acdc;
+}
+
+.nav__list .nav__item {
+  display: inline;
+  position: relative;
+}
+
+nav .nav__list .nav__item .nav__item__link {
+  text-decoration: none;
+  position: relative;
+}
+
+nav .nav__list .nav__item .nav__item__link:active::after {
+  content: "";
+  position: absolute;
+  bottom: 0;
+  border-radius: 10px;
+  left: 0;
+  width: 100%;
+  height: 4px;
+  background-color: #00acdc;
+}
+
+.is-active {
+  color: #00acdc;
+}
+
+nav .nav__list .nav__item .is-active:after {
```

Comment:
Update the selector to use '.nav__item__link.is-active::after' since '.is-active' is a modifier class for the link.